main.tf

```diff
@@ -53,6 +57,14 @@ resource "spacelift_policy" "trigger_dependency" {
   body = file("${path.module}/policies/trigger-dependencies.rego")
 }
 
+# Define the automatic retries trigger policy that allows automatically restarting the failed run
+resource "spacelift_policy" "trigger_retries" {
+  type = "TRIGGER"
+
+  name = "Failed Run Automatic Retries Trigger Policy"
+  body = file("${path.module}/policies/trigger-retries.rego")
+}
+
```

Comment:
2 big concerns about this:
1. What prevents this from creating an infinite loop?
2. We previously decided (see #19) that not only do we not want failed plans to automatically retry, we want failed plans to block automatic plans of future PRs until they have received manual attention.

@osterman Do we have wires crossed somewhere?

Reply:
we had a Jira ticket to add automatic retries, but we definitely need to discuss it

Reply:
added a bool variable to enable/disable this (disabled by default)